python/pylintrc

```diff
@@ -0,0 +1,555 @@
+# NOTE:  Any line with CHANGED: describes something that we changed from the
+# default pylintrc configuration.
+
+[MASTER]
```

Comment:
As of pylint 3.0, `[MASTER]` is deprecated in favor of `[pylint.main]`, which should work even with older versions (apparently all the section headings should start with `pylint.`)

Reply:
I grabbed the latest pylinrc from their upstream git and replaced what I had with that, no system updating needed.